### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by JimmyRomero**

### CODE REVIEW

**Comment by JimmyRomero** [comment]
@carledriss Ready to code review

File: src/main/java/main.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+/**
+ * Created by JimmyRomero on 8/25/2016.
+ */
+public class main {
```
Comment (carledriss) [inline src/main/java/main.java]: Remove this unnecessary class

File: src/test/java/testSquare.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+public class testSquare {
+
+    public static final double DELTA = 0.001;
+
+    private Shape shape;
+
+    @Before
+    public void setUp() {
+        int side = 4;
```
Comment (carledriss) [inline src/test/java/testSquare.java]: It should be final

File: src/test/java/testCircle.java [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+ * Test for Circle {@link Circle}
+ */
+
+public class testCircle {
+
+    private static final double DELTA = 0.001;
+
+    private Shape shape;
+
+    final int radio = 2;
```
Comment (carledriss) [inline src/test/java/testCircle.java:16]: It should be inside of setUp method

**Comment by carledriss** [comment]
@JimmyRomero 
It is missing the implementation for Rectangle

File: src/main/java/Shape.java [hunk 4]
```diff
@@ -0,0 +1,19 @@
+/**
+ * Created by JimmyRomero on 8/24/2016.
+ */
+public abstract class Shape {
```
Comment (carledriss) [inline src/main/java/Shape.java]: Change it for an interface.

File: src/main/java/Triangle.java [hunk 5]
```diff
@@ -0,0 +1,29 @@
+/**
+ * Created by JimmyRomero on 8/25/2016.
+ */
+public class Triangle extends Shape {
```
Comment (carledriss) [inline src/main/java/Triangle.java]: It is missing the implementation for Rectangle

**Comment by carledriss** [comment]
@JimmyRomero 
Let me know when you have fixed the observations

**Comment by JimmyRomero** [comment]
@carledriss 
Observations fixed.

File: src/main/java/Square.java [hunk 6]
```diff
@@ -0,0 +1,27 @@
+/**
+ * Created by JimmyRomero on 8/24/2016.
+ */
+public class Square implements Shape {
```
Comment (carledriss) [inline src/main/java/Square.java]: In fact Square is a Rectangle with equals sides.
Refactor this class.

**Comment by JimmyRomero** [comment]
@carledriss 
Square class refactored, ready to code review.

**Comment by carledriss** [comment]
Looks good to me!

**Pull request merged**